README.markdown

```diff
@@ -11,21 +11,20 @@ Installation:
 
 Usage:
 
-	upstream backend {                                                                              
-		jdomain www.baidu.com; #port=80                                                             
-		#jdomain www.baidu.com port=8080; #port=8080
+	upstream backend {
+		refreshed_hostname www.baidu.com;
```

Comment:
I think the directive could just be `hostname` with refreshed being in the module name just to call out the new behavior